rootfs/etc/profile.d/dev-mode.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+if [ "${DEV}" == "true" ]; then
+    echo "Installation of Ansible dependencies for the Development mode..."
```

Comment:
This should only happen if `/conf/ansible/Makefile` exists. E.g. `geodesic` is used for all kinds of projects, some of which do not use `ansible`